support-frontend/assets/components/headers/links/links.jsx

```diff
@@ -31,73 +35,110 @@ const links: HeaderNavLink[] = [
     href: routes.showcase,
     text: 'Support',
     trackAs: 'showcase',
+    internal: true,
   },
   {
     href: routes.recurringContribCheckout,
     text: 'Contributions',
     trackAs: 'contributions',
     additionalClasses: 'component-header-links__li--show-on-tablet',
+    internal: true,
   },
   {
     href: routes.subscriptionsLanding,
     text: 'Subscriptions',
     trackAs: 'subscriptions',
+    internal: true,
   },
   {
     href: routes.digitalSubscriptionLanding,
     text: 'Digital',
     trackAs: 'subscriptions:digital',
+    internal: true,
   },
   {
     href: routes.paperSubscriptionLanding,
     text: 'Paper',
     trackAs: 'subscriptions:paper',
     include: [GBPCountries],
+    internal: true,
   },
   {
     href: routes.guardianWeeklySubscriptionLanding,
     text: 'Guardian Weekly',
     trackAs: 'subscriptions:guardianweekly',
+    internal: true,
   },
   {
     href: getPatronsLink('support-header'),
     text: 'Patrons',
     trackAs: 'patrons',
     opensInNewWindow: true,
+    internal: false,
   },
 ];
 
 
+function internationalisationID(countryGroupId: ?CountryGroupId = null): ?string {
+
+  if (countryGroupId != null) {
+    const group = countryGroups[countryGroupId];
+
+    if (typeof group !== 'undefined' && group != null) {
```

Comment:
Could replace this if with
return group ? group.supportInternationalisationId || null;